### PR TITLE
Getting started page content for devs; Content TOC

### PR DESCRIPTION
**Pull request by libertymayc**
Table of contents/layout for Getting Started pages + content for devs

### CODE REVIEW

File: site/docs/getting-started/dev.mdx [hunk 1]
```diff
@@ -1,11 +1,147 @@
...
+
+TABLE
+
+### 3. Integrate Salt into your app
+
+To integrate Salt into your project, import the CSS theme and ToolkitProvider component—and then wrap your application in it.
+
+For example, in a typical React project, you can do this in your src/index.js as follows:
+
+```ts
```
Comment (joshwooding) [inline site/docs/getting-started/dev.mdx:96]:
```suggestion
```tsx
```

File: site/docs/getting-started/dev.mdx [hunk 2]
```diff
@@ -1,11 +1,147 @@
...
+);
+```
+
+### 4. Import components
+
+Congratulations, you're now ready to use our UI components. You can now import them from the `@jpmorganchase/uitk-core` package and use them in your JSX code.
+
+For example:
+
+```ts
```
Comment (joshwooding) [inline site/docs/getting-started/dev.mdx:122]:
```suggestion
```tsx
```

File: site/docs/getting-started/dev.mdx (hunk 1 above)
Comment (joshwooding) [reply site/docs/getting-started/dev.mdx:96]: Not sure, if docusarus supports it but generally you would use tsx here

Comment (libertymayc) [reply site/docs/getting-started/dev.mdx]: yeah that works :)

File: site/docs/getting-started/design.mdx [hunk 3]
```diff
@@ -1,15 +1,18 @@
 ---
 sidebar_label: Design
+hide_table_of_contents: true
 ---
 
+import Content from "../../src/components/content/Content.tsx";
+
 # Getting Started for designers
```
Comment (shey-v) [inline site/docs/getting-started/design.mdx:8]:
```suggestion
# Designing with Salt
```

Comment (libertymayc) [reply site/docs/getting-started/design.mdx:8]: the design page will be done separately :)

File: site/src/css/custom.css [hunk 4]
```diff
@@ -146,17 +167,65 @@
...
+  color: var(--uitk-text-primary-foreground);
+  font-size: var(--uitk-text-fontSize);
+  display: flex;
+  flex-direction: row;
+}
+
+.docs-toc-container {
+  --ifm-link-color: var(--uitk-text-secondary-foreground);
+  height: 100%;
+  position: sticky;
```
[the remark this answers is not in the document]
Comment (james-nash) [reply site/src/css/custom.css:186]: Hmm.. on narrower viewports, Docusaurus swaps the right-hand TOC links for an accordion-like thing at the top of the page that shows the TOC links when you expand it. You can see it in action on their own docs here (just try resizing your browser window): https://docusaurus.io/docs

We seem to have broken this somehow. When I view this PR's preview in a narrow browser window the right-hand TOC links remain visible and thus the main page content gets squished into a narrow column.

Can we please restore Docusaurus out-of-the-box behaviour. Even if we don't style up it's accordion thingy, it's at least usable on small screens. What we have at the moment isn't.

In terms of design, my ask would be to then sit down together with @shey-v and try to make Docusaurus' out-of-the-box, small-screen TOC thing blend in better with our look and feel. Hopefully a little bit of CSS to apply appropriate colors and spacing will suffice.

We don't have the time to design and build our own, completely bespoke responsive behaviour for this.

(We can always create tasks on the backlog to make something more bespoke later on)

File: site/docs/getting-started/dev.mdx [hunk 5]
```diff
@@ -1,15 +1,242 @@
...
+For further details and alternative methods, refer to the instructions on Google Fonts tear-out.
+
+**Use FontSource**
+
+You install FontSource’s npm packages and then import their CSS into your application. This will include the necessary web font files into your build so that you can self-host them.
+
+First, install the required NPM packages:
+
+```
+npm install –save @fontsource/open-sans @fontsource/pt-mono
```
Comment (joshwooding) [inline site/docs/getting-started/dev.mdx:75]:
```suggestion
npm install --save @fontsource/open-sans @fontsource/pt-mono
```

File: site/docs/getting-started/dev.mdx [hunk 6]
```diff
@@ -1,15 +1,242 @@
...
+There are three dependencies that you need to add to your project:
+
+- `@jpmorganchase/uitk-core` contains production-ready UI components.
+- `@jpmorganchase/uitk-theme` contains CSS files that apply Salt's default theme.
+- `@jpmorganchase/uitk-icons` contains SVG-based icons.
+
+Use this npm command to install all three dependencies in one go:
+
+```
+npm install --save @jpmorganchase/uitk-core @jpmorganchase/uitk-theme @jpmorganchase/uitk-icons
```
Comment (joshwooding) [inline site/docs/getting-started/dev.mdx:44]: npm no longer needs --save

File: packages/theme/css/characteristics/text.css [hunk 7]
```diff
@@ -24,12 +24,12 @@
   /* H3 */
   --uitk-text-h3-fontWeight: var(--uitk-typography-fontWeight-semiBold);
   --uitk-text-h3-fontWeight-small: var(--uitk-typography-fontWeight-regular);
-  --uitk-text-h3-fontWeight-strong: var(--uitk-typography-fontWeight-extraBold);
+  --uitk-text-h3-fontWeight-strong: var(--uitk-typography-fontWeight-bold);
```
Comment (joshwooding) [inline packages/theme/css/characteristics/text.css:27]: Are these changes intentional? If so they should be in a separate PR

Comment (libertymayc) [reply packages/theme/css/characteristics/text.css:27]: yes, was planning to :)

**Review by JoanaMMoreira — DISMISSED** [review]

**Review by JoanaMMoreira — APPROVED** [review]

**Review by JoanaMMoreira — APPROVED** [review]
